.github/workflows/ci-workflow.yml

```diff
@@ -225,8 +225,13 @@ jobs:
 
   clang-formatting-check:
     name: clang-format check
-    runs-on: kuzu-self-hosted-testing
+    runs-on: ubuntu-22.04
     steps:
+      - name: Install clang-format
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y clang-format-11
```

Comment:
Are we delaying clang format 17 upgrade?

Reply:
clang-format is not backward compatible. If we use clang-format-17 we need to also edit `.clang-format` to achieve the same format. Since I am not familiar with clang-format, I did not handle it in this PR.